internal/pkg/agent/cmd/install.go

```diff
@@ -135,6 +147,9 @@ func installCmd(streams *cli.IOStreams, cmd *cobra.Command, args []string) error
 			}
 		}
 		if token == "" {
+			if nonInteractive {
+				return fmt.Errorf("missing required --enrollment-token")
```

Comment:
[Suggestion]
same as above

Reply:
```suggestion
  return fmt.Errorf("missing required --enrollment-token argument used to enroll the agent")
```
I've pushed the commits with the following changes, matching the format of the messages above